src/Common/Commands.ScenarioTest/packages.config

```diff
@@ -3,8 +3,9 @@
   <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
   <package id="Microsoft.Azure.Common" version="2.0.2" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Extensions" version="0.14.0-preview" targetFramework="net45" />
```

Comment:
Will remove - I actually didn't add it and found it was automagically added by VS - maybe there is something broken somewhere?